features/support/creation_page.rb

```diff
@@ -0,0 +1,41 @@
+require 'page-object'
+
+class CreationPage
+  include PageObject
+
+  page_url "http://localhost:4567/create"
+
+  text_field(:title, :id => "post-title")
+  text_field(:author, :id => "post-author")
+  text_area(:content, :id => "post-body")
+  button(:submit, :id => "submit-post")
+  button(:home, :id => "navigate-home")
+
+  def create_valid_post
+    self.title = "My Blog Title"
+    self.content = "This is my post's content."
+    self.submit
+  end
+
+  def create_duplicate_posts
+    create_valid_post
+    goto
+    create_valid_post
+  end
+  
+  def enter_title
+    self.title = "title"
+  end
+  
+  def enter_body
+    self.content = "body"
+  end
+  
+  def enter_author
+    self.author = "author"
+  end
+  
+  def go_home

```

Comment:
This is an unnecessary level of abstraction. All this method does is call the `home` method, so why not just call the `home` method directly?